### ISSUE TITLE
Sitemap 'default' could not be found

### ISSUE THREAD

**Issue by rbaindourov** [openhab/openhab#4451]
I see some chatter about this, but no quick solution is popping at me.
Can someone help me please?

Using Raspberry PI3, Zstick and a wall plug.

I just installed open-zwave and verified that my zstick is on /dev/ttyACM0 by connecting to with 
`sudo ./MinOZW /dev/ttyACM0`

I went into my openhab config and setup the zwave binding and dropped the zwave addon into the addons folder.

I ran  `sudo ./start_debug.sh` and I got a 500 error when pointing my browser at  `localhost:8080`

## Expected Behavior
Actually I don't know what I should see, I just know its not supposed to a be 500 error!
I want to see a control panel for the zstick so I can manipulate the devices I have.

## Current Behavior
I see a 500 error from openhab.app.

## Possible Solution
I'm open to suggestions, I don't understand what I am supposed to do before I point my browser at openhab. 

## Steps to Reproduce (for bugs)
1. sudo ./start_debug.sh
2. point browser to localhost:8080

## Context
Totally blocked me. 
One hair went gray fizzled and died, screaming all the way down to the floor as it fell from my head. 

## Your Environment
Raspberry PI3 with Raspbian, Zstick, wall plug, open-zwave

**Comment by 9037568** [comment]
What version of OpenHAB are you using?  What did you name your sitemap?

Does the rest of your 500 error says this?
>Problem accessing /openhab.app. Reason:
>
>    Sitemap 'default' could not be found

If it does, you need to use a URL like this one:
>http://[openHABaddress or hostname]:8080/openhab.app?sitemap=[name of your sitemap]

**Comment by 9037568** [comment]
Duh. I should have read the title of the post.

**Comment by rbaindourov** [comment]
Hi,

I am on a Raspbian install on a PI3. So I am using the linux version. :) 

I'm sorry, I have some more questions.
The sitemap is just a website sitemap, right?
I'm assuming this is some weird glitch that the web server needs this to render the layout?
Or am I wrong, and this 'sitemap' is actually a list of devices? 
Or a config file of some kind?

Maybe I am not clued in on the terminology here, because its very peculiar that a web app should fail for just a simple sitemap, which is usually only there for web crawlers. 


So If I just stick a blank file, will the 500 error go away?
Or what is the format? Do I need to install the demo or some sort of layout? It seems like a UI layer is missing.

And what role does HABmin2 play in all this?

I was about to go and install open-zwave-control-panel and then go to try out HABmin.

Thanks a lot for your help!

**Comment by 9037568** [comment]
Thanks for the information. When you start OpenHAB, the log file (under logs/openhab.log) should show the version near the beginning, like so:
>2016-06-06 18:32:06.093 [INFO ] [.o.core.internal.CoreActivator] - openHAB runtime has been started (v1.8.2).

Unfortunately the terminology here is not W3C standard. A sitemap in OpenHAB terms is not related in any meaningful way to a WWW site's sitemap. You can read up about the [OpenHAB sitemap](https://github.com/openhab/openhab/wiki/Explanation-of-Sitemaps) on the wiki.

Basically, if you have a file named "default.sitemap" in the right location, you may be able to use your original http://localhost:8080/openhab.app URL. Otherwise, you'll have to use the form of URL I quoted above.

Also, the [community forum](http://community.openhab.org) is the best place for finding answers to these kinds of questions.

**Comment by rbaindourov** [comment]
My openHAB is v1.8.3

I will spend some time on the forum, thank you for demystifying the sitemap.
Strange programming language for UI definition.
I just want to be able to read data from my wall plug. 
Basic CRUD interface.
Is there nothing existing?

Now I understand what the UI Designer does!
The light has come on.

Thank You. I obviously pointed my browser at it a little early. There is no explicit declaration on the download page that the software ships sans default layout. And the words `Optional` are adorning the UI Designer and other sections. So my mind sort of blackholed the possibility. 

But its clear, I have more work to do, installing a UI.

**Comment by rbaindourov** [comment]
I can't get the Designer to run in my Macbook or Ubuntu. 
I launched the demo for 1.8.3 and its a completely fictitious environment that doesn't exist.
I don't understand, why can't I just get a listing of all the zwave devices?
What do I have to do to get a simple list of all the devices on my zwave network?
I've already been able to do this with Open-ZWave-Control-Panel out of the box.

I tried openHAB version 2 and I installed the ZWave binding, in the paper UI, and then what? I get nothing in the Classic UI.

Where is the documentation on getting a ZWave network list?
